.github/workflows/rust.yml

```diff
@@ -40,6 +40,7 @@ jobs:
     - name: check-features
       run: |
         cargo check --no-default-features --features bit-vec
+        cargo check --no-default-features --features dogfood
```

Comment:
Do we want to keep this optional crate feature? What purpose does it have?

Reply:
That is a good question. I fail to see the reason to keep this as a feature. @ascjones you ok making it non-optional?